### ISSUE TITLE
Telegram Contact Driver Bug

### ISSUE THREAD

**Issue by apm-dev** [mpociot/botman#1202]
- BotMan Version: 2.6.0
- PHP Version: 7.4.7
- Messaging Service(s):
- Cache Driver: Redis

### Description:
There is problem with Telegram contact receiver, it's common that one or more fields of contact be null and then an exception will throw...

### Steps To Reproduce:
In _**TelegramContactDriver**_ class, **_loadMessage_** method:

**Now (with bug)**
```php
$message->setContact(new Contact(
   $this->event->get('contact')['phone_number'],
   $this->event->get('contact')['first_name'],
   $this->event->get('contact')['last_name'],
   $this->event->get('contact')['user_id'],
   $this->event->get('contact')['vcard']
));
```

**Fix**
```php
$message->setContact(new Contact(
   $this->event->get('contact')['phone_number'] ?? null,
   $this->event->get('contact')['first_name'] ?? null,
   $this->event->get('contact')['last_name'] ?? null,
   $this->event->get('contact')['user_id'] ?? null,
   $this->event->get('contact')['vcard'] ?? null
));
```

**Comment by leseulgentil** [comment]
Thanks!

I'm wondering why this kind of solution are not considered in the main source so that when we create new bot from Composer we get a most updated version? 

Please help

**Comment by leseulgentil** [comment]
Do you have an exemple code of how to Ask Phone number?

**Comment by crynobone** [comment]
Fixed via botman/driver-telegram#88 and will be released as `1.6.2` later today
